extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -641,6 +723,54 @@ describes.realWin(
       });
     });
 
+    describe('get amp elements', () => {
+      beforeEach(() => {
+        child.classList.add('i-amphtml-element');
+        toggleExperiment(win, 'visibility-trigger-improvements', true);
```

Comment:
@zhouyx  Any idea why this would not work for the embed case? I'm guessing it has to do with the win I'm passing in.

Reply:
Fix was to use parentRoot.ampdoc.win :)